support/proxy/vwf.example.com/node3.vwf.yaml

```diff
@@ -274,45 +274,6 @@ properties:
   ## @name node3.vwf#vertices
   ## @property
 
-  vertices: 
-    set: |
-      this.logger.info( "WARNING: node3 vertices cannot be set" )
-
-  ## Vertex indices of 3D node
-  ## 
-  ## @name node3.vwf#vertexIndices
-  ## @property
-
-  vertexIndices: 
-    set: |
-      this.logger.info( "WARNING: node3 vertexIndices cannot be set" )
-
-  ## Speed value
-  ## 
-  ## @name node3.vwf#speed
-  ## @property
-
-  speed: 1
-
-  ## Playing value
-  ## 
-  ## @name node3.vwf#playing
-  ## @property
-
-  playing: false
-
```

Comment:
I searched for them yesterday and didn't find any references (except in `control-old.vwf`, which doesn't seem to be used either). They've been replaced by `animationPlaying`, etc. in `animation.vwf`.